src/xtgeoviz/plot/baseplot.py

```diff
@@ -36,6 +36,7 @@ def __init__(self):
         self._tight = False
         self._showok = True
         self._fig = None
+        self._allfigs = []
```

Comment:
i have a weak-feeling that this should maybe be a weakref.

Reply:
I'm not so sure about this either. Hopefully we can come back to this code at some point. But for now, it's just copying the current state of xtgeo.plot to here.

I made an issue for it: https://github.com/equinor/xtgeoviz/issues/30